src/components/Navigation.js

```diff
@@ -11,14 +11,38 @@ import { t } from 'ttag';
 import logo from '../assets/images/hathor-white-logo.png';
 import Version from './Version';
 import ServerStatus from './ServerStatus';
+import { EXPLORER_BASE_URL } from '../constants';
 
+let shell = null;
+if (window.require) {
+  shell = window.require('electron').shell;
+}
 
 /**
  * Component that shows a navigation bar with the menu options
  *
  * @memberof Components
  */
 class Navigation extends React.Component {
+
+  /**
+   * Method called when user clicked on Esxplorer menu
+   *
+   * @param {Object} e Event for the click
+   */
+  goToExplorer = (e) => {
```

Comment:
Could we have an util method for opening web pages? It seems it's needed in some different places.

Reply:
Maybe just an issue to do it later.

Reply:
Looks good now!